index.js

```diff
@@ -1,4 +1,12 @@
-'use strict';
-// we're gonna use strict  mode in all
+try {
```

Comment:
Bad practice, only use the try catch on asynchronous flow, in this file we don't needed